portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -696,13 +696,37 @@ definition {
 	}
 
 	@description = "Verify that a Form can be deleted"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDeleted {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
+	
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+
+		Form.editName(formName = "Form 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
 
-		// TODO LPS-130509 CanBeDeleted pending implementation
+		Form.save();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		AssertElementPresent(
+			key_formName = "Form 1",
+			locator1 = "FormsAdmin#FORMS_TABLE_NAME");
```

Comment:
Hi Marcio,
In order to check the forms name, you can use a macro already created for Forms called `viewForm`.

In this test scenario, it can be used like this:

```
FormsAdmin.viewForm(
			formDescription = "",
			formName = "Form 1");
```

Reply:
@brunofariasdeo requests done!